### ISSUE TITLE
Incorrect business process

### ISSUE THREAD

**Issue by artur-beljajev** [internetee/registry#636]
Creating new bank transaction is weird, since we are not bank. . I guess that feature exists for manual account entry creation. This is anyway confusing for both users and developers.

http://prntscr.com/h7vf7j

**Comment by vohmar** [comment]
this is correct, follows agreed business requirements and process and is by design. 
Description of this ticket is too vague for feature request.

**Comment by artur-beljajev** [comment]
So is this 

> feature exists for manual invoice payment creation. 

true? Perhaps it's better to call it "Bank payments"? I am not sure I understand how a domain registry can create new bank transactions and why. Can you elaborate?
